austrakka/components/field/__init__.py

```diff
@@ -23,6 +23,11 @@ def field_list(out_format: str):
 @field.command('add', hidden=hide_admin_cmds())
 @opt_name(help="Field name")
 @opt_fieldtype()
+@click.option('-d', '--description', 'description',
+              help="This field describes the purpose of the metadata field. "
+                   "It's value is also used for generating XLSX pro forma files.")
+@click.option('--nndss-label', 'nndss_label',
+              help="This National Notifiable Diseases Surveillance System label.")
 @click.option('--colour-nodes', 'colour_nodes', flag_value='viz',
```

Comment:
@viatunimelb typos: "It's value" -> "Its value", and "This National Notifiable Diseases Surveillance System label" -> "The corresponding National Notifiable Diseases Surveillance System label" (only the word this is a typo)